charts/newrelic-infrastructure-beta/templates/NOTES.txt

```diff
@@ -0,0 +1,26 @@
+Your deployment of the New Relic Infrastructure agent is complete. You can check on the progress of this by running the following command:
```

Comment:
I would add "If you have not used `--wait` during the installation of this helm chart" to this sentence. And I would add it to the docs so Helm waits until it is deployed.

This has make me remind of creating a (Helm Chart test)[https://helm.sh/docs/topics/chart_tests/] so we can also tests things like "the secrets you have put are correct" or "the license key is correct, valid and allow to ingest data" (if possible).